test/Microsoft.Sbom.Api.Tests/Config/ConfigSanitizerTests.cs

```diff
@@ -351,4 +352,30 @@ public void ConfigSantizer_Validate_ReplacesBackslashes_Linux(ManifestToolAction
             Assert.IsTrue(config.TelemetryFilePath.Value.StartsWith($"/{nameof(config.TelemetryFilePath)}/", StringComparison.Ordinal));
         }
     }
+
+    [TestMethod]
```

Comment:
You may need to break this down into separate tests to test the allowed range of values